virtio-devices/src/epoll_helper.rs

```diff
@@ -102,8 +127,11 @@ impl EpollHelper {
             };
 
             for event in events.iter().take(num_events) {
+                let ev_events = event.events as i32;
                 let ev_type = event.data as u16;
 
+                error!("event detected! type: {} event:{}", ev_type, ev_events);
```

Comment:
This looks to be a debug aid. It should be either dropped or turned into `debug!`

Reply:
I removed this line.